### DISCUSSION TITLE
Any Benchmark Data or Publications for Island Test Results?

### DISCUSSION THREAD

**Discussion by andrewlyu** [Genesis-Embodied-AI/genesis-world#421] [General]
I’m running the test_rigid_island.py script and noticed very little FPS improvement:
```
True island  | 1 cubes  | <CG: 0>  | 38,318,023.17 fps  | 8192 envs  
False island | 1 cubes  | <CG: 0>  | 36,065,914.07 fps  | 8192 envs  
``` 
Is there any benchmark data or related publications I can refer to for these results? I’d like to better understand the context and expected performance for this test.

Thanks for help!

**Comment by zhouxian** [comment]
No -- 
firsly, that's for internal test and not meant for public usage
secondly, contact island is for improving collision resolving speed when there's more than one collision pairs. so the case with 1 cube is not really affected

**Reply by andrewlyu** [reply]
Thanks! I change n of cubes to 8 and observe a good improvement:
```
True island 	| 8 cubes 	| <CG: 0> 	| 278,399.62 fps 	| 8192 envs
False island 	| 8 cubes 	| <CG: 0> 	| 180,908.29 fps 	| 8192 envs
```
